components/dialog/dialog-fullscreen.js

```diff
@@ -161,10 +170,6 @@ class DialogFullscreen extends LocalizeCoreElement(AsyncContainerMixin(DialogMix
 					height: calc(100% - 1px);
 				}
 
-				div[nested].d2l-dialog-outer {
-					top: 0;
-				}
-
```

Comment:
As far as I can tell, this block and the corresponding one in `dialog.js` don't do anything.  `nested` used to be added as an attribute, but was changed to the `d2l-dialog-outer-nested` class in https://github.com/BrightspaceUI/core/pull/573.  I think these were just missed in the update, and switching to the `class` changed the specificity such that nothing broke and these `media` overrides weren't needed.